app/views/users/show.html.erb

```diff
@@ -7,20 +6,25 @@
 </head>
 <body>
   <header>
-    <h1>Welcome to My Blog</h1>
+    <h1><%= @user.name %> 's Profile</h1>
   </header>
 
-  <nav>
-    <ul>
-      <li><a href="/">Home</a></li>
-    </ul>
-  </nav>
-
-  <main>
-    <h2>User Profile</h2>
-    <!-- Add your user profile content here -->
-  </main>
-
+  <p>Number of Posts: <%= @user.posts.count %></p>
+  
+  <p><strong>Bio:</strong> <%= @user.bio %></p>
+  
+  <h2>Recent Posts:</h2>
+  <% @recent_posts.each do |post| %>
+    <div>
+      <h3><%= post.title %></h3>
+      <p><%= post.text %></p>
+    </div>
+  <% end %>
+    
+  <p><%= link_to "All Posts", user_posts_path(@user) %></p>
+  
+  <li><a href="/">back to home</a></li>
```

Comment:
- [ ] It is nice to keep to design guidelines, kindly match your UI to the given wireframe
in your design :point_down: 
![Screenshot from 2023-11-25 11-47-37](https://github.com/paulshaun-kdtk/blog-posts/assets/66097371/2d464c31-3a9e-44b8-b2a2-22efb2213d76)
given wireframe :point_down: 
![Screenshot from 2023-07-20 22-23-13](https://github.com/paulshaun-kdtk/blog-posts/assets/66097371/a17d1af8-a58d-472a-a8c9-f632f6e75bd4)